scripts/netContain/scripts/contivNet.sh

```diff
@@ -19,6 +19,9 @@ vlan_if_param=""
 control_url=":9999"
 listen_url=":9999"
 
+touch /tmp/restart_netmaster
```

Comment:
This is to give explicit control to the system-test framework to kill the netplugin and netmaster process and start them with custom arguments which is required to test different scenarios. Without this control, the script automatically spawns off the netplugin/netmaster process within 5 sec. Will insert a comment here.